contracts/src/v0.8/dev/automation/2_1/KeeperRegistry2_1.sol

```diff
@@ -293,39 +297,37 @@ contract KeeperRegistry2_1 is KeeperRegistryBase2_1, OCR2Abstract, Chainable, ER
     s_transmittersList = transmitters;
 
     // Set the onchain config
-    OnchainConfig memory onchainConfigStruct = abi.decode(onchainConfig, (OnchainConfig));
-    if (onchainConfigStruct.maxPerformGas < s_storage.maxPerformGas) revert GasLimitCanOnlyIncrease();
-    if (onchainConfigStruct.maxCheckDataSize < s_storage.maxCheckDataSize) revert MaxCheckDataSizeCanOnlyIncrease();
-    if (onchainConfigStruct.maxPerformDataSize < s_storage.maxPerformDataSize)
-      revert MaxPerformDataSizeCanOnlyIncrease();
+    OnchainConfig memory onchainConfig = abi.decode(onchainConfigBytes, (OnchainConfig));
+    if (onchainConfig.maxPerformGas < s_storage.maxPerformGas) revert GasLimitCanOnlyIncrease();
+    if (onchainConfig.maxCheckDataSize < s_storage.maxCheckDataSize) revert MaxCheckDataSizeCanOnlyIncrease();
+    if (onchainConfig.maxPerformDataSize < s_storage.maxPerformDataSize) revert MaxPerformDataSizeCanOnlyIncrease();
```

Comment:
general q. do we see value in having these invariants?

Reply:
down to remove

Reply:
decided to complete in a separate ticket